guides/quickstart.md

```diff
@@ -4,426 +4,172 @@ title: Quickstart
 ---
 
 {% raw %}
-This is a short guide to get you started writing page objects and using them in your acceptance and integration tests.
 
-- [Acceptance tests](#acceptance-tests)
-- [Integration tests](#integration-tests)
+This is a short guide to get you started building powerful UI testing APIs using EmberCLI Page Object.
 
-## Acceptance Tests
+- [Basic Component](#basic-component)
+- [Pages](#pages)
 
-Suppose we have a couple of acceptance tests to test the login page of our site.
+## Basic Component
 
-```js
-test('logs in sucessfully', function(assert) {
-  visit('/login');
-  fillIn('#username', 'admin');
-  fillIn('#password', 'secret');
-  click('button');
-
-  andThen(function() {
-    assert.equal(currentURL(), '/private-page');
-  });
-});
+The primary build block of page objects is a [component](./components). It consists of a [`scope`](./components#scopes), [`attributes`](./components#attributes), custom methods and nested components.
 
-test('shows an error when password is wrong', function(assert) {
-  visit('/login');
-  fillIn('#username', 'admin');
-  fillIn('#password', 'invalid');
-  click('button');
+Assume there is a simple `<SearchForm />` component with a text field and a submit button on it.
 
-  andThen(function() {
-    assert.equal(currentURL(), '/login');
-    assert.equal($.trim(find('.errors').text()), 'Invalid credentials');
-  });
-});
-```
+__Example__
 
-We want to convert these tests to use a page object.
+```html
+<form class="search-form">
+  <input type="search">
+  <button>Search</button>
+</form>
+```
 
-First, we need to create a new page object. For this we'll use one of the generators that comes with the addon.
+Let's generate a page object component definition for it with a help of corresponding generator:
```

Comment:
Let's generate a page object component definition for it with **the** help of **the** following generator: